### PR TITLE
Fix bug when  is null like launch with jenkins

### PR DESCRIPTION
**Pull request by supernana**
I have notice this line could make some troubles with twig cache

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
:+1:

File: Renderer/Adapter/Twig.php [hunk 1]
```diff
@@ -77,7 +77,7 @@ public function __construct(AbstractRenderer $renderer, array $config = [])
         if ($application->isDebugMode() || (isset($config['debug']) && true === $config['debug'])) {
             $this->twig->enableDebug();
             $this->twig->addExtension(new \Twig_Extension_Debug());
-        } elseif ($application->isClientSAPI()) {
+        } elseif ( null !== $application && false === $application->isClientSAPI()) {
```
Comment (eric-chau) [inline Renderer/Adapter/Twig.php]: So you inverted the ``$application->isClientSAPI()``? Is it really what you wanted to do? Also, you test if ``$application`` is not null, wouldn't it be better to do it earlier? As the first ``if`` condition can benefit of it too.

**Comment by eric-chau** [comment]
Outdated PR.
